shopfloor_mobile/static/wms/src/main.js

```diff
@@ -121,6 +122,21 @@ const app = new Vue({
                 this.$storage.set("profile", v);
             },
         },
+        has_set_workstation: function() {
```

Comment:
this is useless IMO. You can simply check `this.$root.workstation`

Reply:
Thanks I removed it